examples/extras/requirements.txt

```diff
@@ -0,0 +1 @@
+google-cloud-language==0.27.0
```

Comment:
FYI the 0.27.0, 0.28.x, and 0.29 releases all had wide-ranging changes to deps, and various problems with deps, unrelated to the issue at hand.